packaging/windows/build_pypi_pkg/generate_win_pypi_pkg.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
could you add more information about the context to the PR description?
also, a sample command.

Reply:
Resolved